pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/ActivityAnswerCopierSql.sql.stg

```diff
@@ -0,0 +1,107 @@
+group ActivityAnswerCopierSql;
+
+getLatestAnswer() ::= <<
+select
+src_a.answer_id,
+src_qt.question_type_code
+from
+answer src_a
+join question as src_q on src_q.question_id = src_a.question_id
+join question_type src_qt on src_qt.question_type_id = src_q.question_type_id
+join activity_instance as src_ai on src_ai.activity_instance_id = src_a.activity_instance_id and src_ai.activity_instance_id = :activityInstanceId
+join study_activity as src_act on src_act.study_activity_id = src_ai.study_activity_id
+join question_stable_code as src_q_code on src_q_code.question_stable_code_id = src_q.question_stable_code_id and src_q_code.stable_id = :questionStableId
+where
+src_a.last_updated_at = (
+	select max(a2.last_updated_at)
+	from
+    answer a2
+    where
+    a2.question_id = src_a.question_id
+    and
+    a2.operator_user_id = src_a.operator_user_id
+    and
+    a2.activity_instance_id = src_ai.activity_instance_id)
+>>
+
```

Comment:
Would it simplify query if instead of `last_update_at` subquery, you  `SORT` by that column and then use `LIMIT 1` to get the first item ?

Reply:
Might be an equivalent query, but it would be more orthodox to have the query arguments be used in the `WHERE` clause rather than as part of the `JOIN` clauses.

Reply:
Is `operator_user_id` needed?

Reply:
Oy, my prehistoric oracle days are coming back to haunt me--order by and limit definitely make this simpler!  I'm not sure about performance, but I suspect that it's a wash.  Gotta do the sort somewhere, right?

Reply:
I don't think we need to restrict this by operator.  We're just copying whatever the latest answer is, regardless of who put it in.  Although now I'm second guessing myself.  Is there a scenario where this would get us into trouble?

Reply:
Will definitely move the args into a `where`.